hack/make/.integration-test-helpers

```diff
@@ -33,6 +33,10 @@ run_test_integration() {
 }
 
 build_test_suite_binaries() {
+	if [ $DOCKER_INTEGRATION_TESTS_VERIFIED ]; then
```

Comment:
The older version of the build system had this variable.
I kept the variable name for compatibility.